src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryFetchFunction.java

```diff
@@ -413,7 +410,8 @@ private RepositoryDirectoryValue tryGettingValueUsingVendoredRepo(
   }
 
   /* Determines whether we should use the cached repositories */
-  private boolean shouldUseCachedRepos(Environment env, Rule rule) throws InterruptedException {
+  private boolean shouldUseCachedRepos(Environment env, RepoDefinition repoDefinition)
```

Comment:
Can we rename this to singular form and update the comment?

Reply:
renamed to `shouldUseCachedRepoContents`